targets/XPlatCppSdk/source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -30,6 +94,12 @@ namespace PlayFab
         QoSResult PlayFabQoSApi::GetQoSResult(unsigned int numThreads, unsigned int timeoutMs)
         {
             QoSResult result(move(GetResult(numThreads, timeoutMs)));
+
+            if (result.errorCode != QoSErrorCode::Success)
+            {
+                return result;
```

Comment:
>return result; [](start = 16, length = 14)

Is it intentional that we don't want to send failures to PlayFab?

Reply:
Yeah, this is caused by either player not logged in or PlayFab doesn't reply with QosServers list. We shouldn't spam the event stream with those failures.